homeassistant/components/sensor/websocket_api.py

```diff
@@ -0,0 +1,33 @@
+"""The sensor websocket API."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DEVICE_CLASS_UNITS
```

Comment:
Note: just because a device class has units doesn't mean that a converter exists between the units.
Should there be a similar `device_class_convertible_units` WS command?

Reply:
Thanks @epenet, the point of this command is indeed to list convertible units 👍